verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -151,7 +210,7 @@ Output: Produces Indexing Facts for kythe (http://kythe.io).
     files_names.push_back(filename);
   }
 
-  int exit_status = verilog::kythe::ExtractFiles(
-      files_names, verible::file::Dirname(args[1]));
-  return exit_status;
+  std::vector<absl::Status> status = verilog::kythe::ExtractFiles(
```

Comment:
"statuses" (I'll fix.)

I'll see if I can print diagnostics to `std::cerr` without breaking tests.